cyphernodeconf_docker/templates/bitcoin/bitcoin.conf

```diff
@@ -32,6 +35,13 @@ test.wallet=xpubwatching01.dat
 test.wallet=spending01.dat
 test.wallet=ln01.dat
 test.rpcbind=0.0.0.0
+<% } else if ( net === 'regtest' ) { %>
+regtest.wallet=watching01.dat
+regtest.wallet=xpubwatching01.dat
+regtest.wallet=spending01.dat
+regtest.wallet=ln01.dat
+regtest.rpcbind=0.0.0.0
+regtest.rpcport=18443
```

Comment:
Why do you specify the port number for regtest but not for the other chains?  Should we just use the default one?  Or specify them for testnet and mainnet?

Reply:
whats the default regtest port? :D